src/new/wallet.js

```diff
@@ -205,6 +205,22 @@ class HathorWallet extends EventEmitter {
     return historyTransactions;
   }
 
+  /**
+   * Get a transaction data from the wallet
+   *
+   * @param {String} id Hash of the transaction to get data from
+   *
+   * @return {Object} Data from the transaction to get. Can be null if the wallet does not contain the tx.
```

Comment:
Maybe add here which pieces of data are provided?

Reply:
Done